CONTRIBUTING.md

```diff
@@ -613,14 +603,14 @@ existence of those docs.
 
 ### Reporting security issues
 
-If you have found an issue in an Akka project that might have security
-implications, you can report it to <security@lightbend.com>. We will make
+If you have found an issue in an Pekko project that might have security
+implications, you can report it to <???>. We will make
```

Comment:
link to Security.md - https://github.com/apache/.github/blob/main/.github/SECURITY.md

Reply:
Done - please review.